src/main/java/bl4ckscor3/mod/isamodbyvazkii/IsAModByVazkii.java

```diff
@@ -37,12 +37,13 @@ public void postInit(FMLPostInitializationEvent event)
 		{
 			String authorList = modList.get(modid).getMetadata().getAuthorList();
 
-			if(authorList.toLowerCase().contains("vazkii")) {
+			if(authorList.toLowerCase().contains("vazkii")) 
 				MODS.put(modid, modList.get(modid).getName());
-			}
 
-			if(modList.get(modid) instanceof FMLModContainer) {
-				if(modList.get(modid).getMetadata().getAuthorList().toLowerCase().contains("vazkii")) {
+			if(modList.get(modid) instanceof FMLModContainer) 
+      {
+				if(authorList.toLowerCase().contains("vazkii")) 
+        {
```

Comment:
Could you indent this one and these braces correctly, please?

Reply:
right... Vim did an oof